.circleci/config.yml

```diff
@@ -63,13 +50,51 @@ jobs:
               python integration.py compare \
                 --report-path test-reports \
                 $upstream_short_id $head_short_id
-      - store_artifacts:
+      - store_artifacts: &store_artifacts
           path: /app/test-reports
           destination: /app/test-results
+      - save_cache: &save_cache
+          paths:
+            - ~/resources
+          key: mps-integration-data-v1-{{ epoch }}
+  integrate-clean:
+    docker:
+      - image: mozilla/edge-validator:1.3
```

Comment:
```suggestion
      <<: *edge_validator
```

Reply:
Pushed an update that refactors this to an executor.